docs/Card/integration.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Pay with Card Custom Integration
+keywords: 
+contentType: docs
+productStatus: current
+apiVersion: TODO
+sdkVersion: TODO
+---
+
+# Pay with Card Custom Integration
+
+Follow these steps to add Card payments.
+
+1. [Know before you code](#know-before-you-code)
+1. [Add Card Payments](#add-card-payments)
+1. [Test and go live](#test-and-go-live)
+
+## Know before you code
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID and generate an access token. 
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
+For initial setup, the `curl` commands below can be used in place of a server SDK.
+
+## Add Card Payments
+
+Accept card payments with PayPal Payments SDK using your own UI.
+
+### 1. Add the Payments SDK  to your app
+
+#### Swift Package Manager
+
+To add the PayPal package to your Xcode project, select _File > Swift Packages > Add Package Dependency_ and enter https://github.com/paypal/iOS-SDK as the repository URL. Tick the checkboxes for the specific PayPal libraries you wish to include.
```

Comment:
In attempt to avoid documenting other tools, maybe consider pointing them to the Xcode docs on adding a dependencies? https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app 

Doing that you could then get something like:
```
In Xcode, add the [package dependency](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) and enter https://github.com/paypal/iOS-SDK as the repository URL. Tick the checkboxes for the specific PayPal libraries you wish to include.
```

Reply:
Agreed - then if Xcode has an update and the location of this changes, we won't have to change the docs